Configurations/10-main.conf

```diff
@@ -804,7 +804,13 @@ my %targets = (
         perlasm_scheme   => "64",
         multilib         => "64",
     },
-
+    
+    # riscv64 below refer to contemporary RISCV Architecture
+    # specifications,
+    "linux64-riscv64" => {
+        inherit_from     => [ "linux-generic64"],
+        perlasm_scheme   => "linux64",
+    },
```

Comment:
There should be empty line after this one to separate it from different architecture.

Reply:
done